src/utils/issue.ts

```diff
@@ -8,19 +8,14 @@ export function isParentIssue(body: string) {
 }
 
 export async function getAssignedIssues(context: Context, username: string): Promise<Issue[]> {
-  const payload = context.payload;
+  const { payload } = context;
 
   try {
-    return await context.octokit.paginate(
-      context.octokit.issues.listForRepo,
-      {
-        owner: payload.repository.owner.login,
-        repo: payload.repository.name,
-        state: ISSUE_TYPE.OPEN,
-        per_page: 100,
-      },
-      ({ data: issues }) => issues.filter((issue: Issue) => !issue.pull_request && issue.assignee && issue.assignee.login === username)
-    );
+    return (await context.octokit.search
+      .issuesAndPullRequests({
+        q: `is:open assignee:${username} org:${payload.repository.owner.login}`,
```

Comment:
```suggestion
        q: `is:open is:issue assignee:${username} org:${payload.repository.owner.login}`,
```
to match only issues

Reply:
My mistake I just double-checked the [docs](https://docs.github.com/en/search-github/searching-on-github/searching-issues-and-pull-requests) too to correct myself, I used the UI to test the query and must have been in the wrong list as it threw an error so I avoided it here

That endpoint is so OP it's crazy